docs/usage/frameworks/litestar.rst

```diff
@@ -397,12 +396,11 @@ If you use a different path than `./migrations`, be sure to also set this in you
         ),
     )
 
-And then run the following:
+And then run the following to initialize the migrations directory:
 
-.. code-block:: bash
+.. code-block:: shell-session
 
-    # Initialize migrations directory
```

Comment:
i removed the comments to so that the copy button only copies the shell commands